src/styles/index.scss

```diff
@@ -1,3 +1,141 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
```

Comment:
The font-family declaration includes 'Roboto' with a fallback of 'serif'. Since 'Roboto' is a sans-serif font, it would be more appropriate to use a sans-serif fallback, such as 'sans-serif'.